src/definitions/behaviors/form.js

```diff
@@ -1818,6 +1819,74 @@ $.fn.form.settings = {
       ;
     },
 
+    // applies rule if another field's value satisfies a condition
+    condition: function(value, ancillary, $module) {
+      let params = ancillary.split(',');
+      if(params.length !== 3) {
+        return false;
+      }
+
+      let
+          identifier  = params[0].trim(),
+          condition   = params[1].trim(),
+          rule        = params[2].trim()
+      ;
+      if(rule.includes('condition')) {
+        // Don't support nested condition rules
+        return false;
+      }
+
+      var
+          fieldValue,
+          fieldElement
+      ;
+      if((fieldElement = $module.find('[data-validate="'+ identifier +'"]')).length > 0 ) {
```

Comment:
I think you should use `module.get.value(identifier)` to fetch the fieldValue instead, because it will take care of all possible form types like checkbox, radio or calendar which need special handling to get the value.

Reply:
Hi @lubber-de , thank you for your advice! I've updated both PR and JSFiddle.